src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -100,14 +58,14 @@ def test_WINNF_FT_S_FAD_1(self):
             'lowFrequency': 3640000000.0,
             'highFrequency': 3650000000.0
         }
-        request = {'grantRequest': [grant_a, grant_c]}
+        grants_to_request = [grant_a, grant_c]
+        grant_request = {'grantRequest': grants_to_request}
         # Send grant requests
         grant_response = self._sas.Grant(request)['grantResponse']
         grant_ids = []
         for resp in grant_response:
```

Comment:
We should probably assert that the response code is zero just to be safe (if the grant request failed for some reason, it will be easier to debug with an extra assert here).